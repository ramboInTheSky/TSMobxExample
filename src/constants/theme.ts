
export const variables = Object.freeze({
    columnSize: 80,
    componentBackground: '#ececec',
    componentBackgroundClear: '#fff',
    spacer: '2px solid #999',
    gutter: '1rem',
    indentation: '0.3rem',
    lineHeightComponentTitle: '2',
    headerFontSize: '1.2rem',
    archetypeColor: '#a4d8f1',
    tagColor: '#b4dec7',
    roomColor: '#eff180',
    primaryColor: '#b53f79',
    visitDateFromColor: '#999',
    visitDateToColor: '#999',
    dangerColor: '#f72b2b',
})