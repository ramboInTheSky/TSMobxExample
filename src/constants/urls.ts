import apiVersion from './api_versions'

export const urls = Object.freeze({
    profiles: `/profiles.json`,
    visits: `https://csi-dev.thecollective.com/${apiVersion.visit}visit`,
    archetypes: `/archetypes.json`,
    lockGroups: `get_lock_groups.json`,
})
export default urls