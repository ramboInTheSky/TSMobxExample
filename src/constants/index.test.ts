import { variables } from './theme'
import { urls } from './urls'

describe('<variables />', () => {
	it('renders correctly', () => {
		expect(variables).toMatchSnapshot()
	})
})
describe('<urls />', () => {
	it('renders correctly', () => {
		expect(urls).toMatchSnapshot()
	})
})
