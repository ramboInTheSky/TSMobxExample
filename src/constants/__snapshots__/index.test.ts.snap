// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<urls /> renders correctly 1`] = `
Object {
  "archetypes": "/archetypes.json",
  "lockGroups": "get_lock_groups.json",
  "profiles": "/profiles.json",
  "visits": "https://csi-dev.thecollective.com/ac-api/visit",
}
`;

exports[`<variables /> renders correctly 1`] = `
Object {
  "archetypeColor": "#a4d8f1",
  "columnSize": 80,
  "componentBackground": "#ececec",
  "componentBackgroundClear": "#fff",
  "dangerColor": "#f72b2b",
  "gutter": "1rem",
  "headerFontSize": "1.2rem",
  "indentation": "0.3rem",
  "lineHeightComponentTitle": "2",
  "primaryColor": "#b53f79",
  "roomColor": "#eff180",
  "spacer": "2px solid #999",
  "tagColor": "#b4dec7",
  "visitDateFromColor": "#999",
  "visitDateToColor": "#999",
}
`;
